wpilibc/src/main/native/include/frc/livewindow/LiveWindow.h

```diff
@@ -20,14 +20,16 @@ class LiveWindow {
   LiveWindow(const LiveWindow&) = delete;
   LiveWindow& operator=(const LiveWindow&) = delete;
 
-  std::function<void()> enabled;
-  std::function<void()> disabled;
+  static std::function<void()> enabled;
+  static std::function<void()> disabled;
```

Comment:
These static variables need to also be declared in the .cpp file, e.g.
`std::function<void()> LiveWindow::enabled;`

Reply:
They are declared in the cpp file (and initialized to a nullptr)

I don't think we can put them in the impl because they get changed from within external classes.